ads/_config.js

```diff
@@ -57,6 +58,12 @@ export const adPreconnect = {
     'https://s.yieldmo.com',
     'https://ads.yieldmo.com',
   ],
+  revcontent: [
+    'https://trends.revcontent.com',
+    'https://cdn.revcontent.com',
+    'https://img.revcontent.com',
+    'https://labs-cdn.revcontent.com',
```

Comment:
You can remove the `'https://labs-cdn.revcontent.com'` entry here, as all entries in the prefetch list also always get a get a preconnect.